interface/wx/sysopt.h

```diff
@@ -81,6 +81,13 @@
         using it, i.e. this has the same effect as calling
         wxApp::MSWEnableDarkMode(). If set to 2, use dark mode unconditionally,
         as if this function were called with wxApp::DarkMode_Always argument.
+    @flag{msw.native-dialogs-pmdpi}
+        If set to 1, native win32 dialogs (like the font and colour pickers)
+        will always be shown as per-monitor DPI aware (when enabled in the manifest).
+        By default, these are only per-monitor DPI aware when all monitors have
```

Comment:
Sorry, but what does "By default, these are only per-monitor DPI aware when all monitors have the same DPI, ..." mean?

Is it factually any different then "By default, these are not per-monitor DPI aware."?

Reply:
Indeed, if you don't support moving between monitors, they are not per-monitor DPI aware. I'll rephrase.